utils/scripts/team.ts

```diff
@@ -0,0 +1,136 @@
+
+    if(updateError){
+        console.log(updateError.message);
+        return updateError;
+    }
+
+    return null;
+}
+
+export const demoteAdmin = async(userID: string, orgID: number) => {
+    const supabase = createClient();
+
+    const {data: admins, error } = await supabase.from('organizations').select('admins').eq('id', orgID);
+    if(error){
+        console.log(error.message);
+        return error;
+    }
+
+    const adminList = admins[0].admins;
+    let newAdminList = adminList;
+    if(adminList.includes(userID)){
+        newAdminList = adminList.filter(function(id){return id !== userID});
+    }
+
+    const {error: updateError} = await supabase.from('organizations').update({admins: newAdminList}).eq('id', orgID);
+
+    if(updateError){
+        console.log(updateError.message);
+        return updateError;
+    }
+
+    return null;
+}
+
+export const removeMember = async(userID: string, orgID: number) => {
+    const supabase = createClient();
+
+    const {data: admins, error } = await supabase.from('organizations').select('admins').eq('id', orgID);
+    if(error){
+        console.log(error.message);
+        return error;
+    }
+
+    const adminList = admins[0].admins;
+    let newAdminList = adminList;
+    // need to remove user from admin first
+    if(adminList.includes(userID)){
+        newAdminList = adminList.filter(function(id){return id !== userID});
+        await supabase.from('organizations').update({admins: newAdminList}).eq('id', orgID);
+    }
+
+    const {error: updateError} = await supabase.from('profiles').update({FK_organizations: null}).eq('id', userID);
+    if(updateError){
+        console.log(updateError.message);
+        return updateError;
+    }
+
+    return null;
+}
+
+export const changeOwners = async(userID: string, orgID: number) => {
+    const supabase = createClient();
+
+    // must remove user from admins first before making owner
+    const error = await demoteAdmin(userID, orgID);
+    if(error){
+        console.log(error)
+        return error;
+    }
+
+    const {error: updateError} = await supabase.from('organizations').update({owner: userID}).eq('id', orgID);
+    if(error){
+        console.log(updateError.message);
+        return updateError;
+    }
+    return null;
+}
+
+export const reloadTeam = async(orgID: number) => {
+    const supabase = createClient();
+    const ownerID = await getOrgOwner(orgID);
+    if(ownerID == null){
+        return [];
+    }
+
+    const {data: adminData, error: errorAdmin} = await supabase.from('organizations').select('admins').eq('id', orgID);
+    if(errorAdmin){
+        console.log(errorAdmin.message);
+        return [];
+    }
+    const admins = adminData[0].admins;
+
+    const { data, error } = await supabase.from('profiles').select('id, full_name, email').eq('FK_organizations', orgID);
+    if(error){
+        console.log(error.message);
+        return [];
+    }
```

Comment:
How will we distinguish whether a returned empty array means there was an error on the server side or there is just no data for the array to be filled with? We should be either be throwing an error message or null for the client to catch and handle it.

Reply:
Whoops this function is unused and didn't flush it out since i didn't end up using it